### ISSUE TITLE
Getting strange results when using pretrained settings

### ISSUE THREAD

**Issue by DinosaurusAlbert** [eraserNut/MTMT#41]
Hi, I am new to image segmentation. Your method seems very promising but when I try loading your pretrained weights I get strange results. It does not seem to detect the shadows. I've downloaded the iter10000.pth file and load it using 
`net = build_model()
checkpoint = torch.load('iter_10000.pth',map_location=torch.device('cpu'))
net.load_state_dict(checkpoint)`
then I run 
`xxx=net(img_var)[0][0]
res = torch.sigmoid(xxx)`
However, the masked images it is outputting are off. Please compare the ground truth image, input, and my result. It is doing some sort of segmentation, but it is not accurately finding the shadows. I am sure I am loading the wrong model or doing something wrong in the settings. Do you have any pointers to what I am doing wrong? Thanks very much!

![13191932324_cb403f5c0e jpg_ground_truth](https://github.com/eraserNut/MTMT/assets/139976395/3482d7ff-439e-49b0-a9ea-ca27a504ec2c)
![13191932324_cb403f5c0e jpg_original](https://github.com/eraserNut/MTMT/assets/139976395/43cb1dc6-fcd4-4d66-8c25-3023fb275dff)
![13191932324_cb403f5c0e jpg_mask2ed](https://github.com/eraserNut/MTMT/assets/139976395/6eeaaf7d-1172-4ff7-aca8-d8cc9b0a6bb4)
